x-pack/plugins/rollup/index.js

```diff
@@ -28,6 +29,20 @@ export function rollup(kibana) {
       managementSections: [
         'plugins/rollup/crud_app',
       ],
+      uiSettingDefaults: {
+        [CONFIG_ROLLUPS]: {
+          name: i18n.translate('xpack.rollupJobs.rollupIndexPatternsTitle', {
+            defaultMessage: 'Enable rollup index patterns',
+          }),
+          value: true,
+          description: i18n.translate('xpack.rollupJobs.rollupIndexPatternsDescription', {
+            defaultMessage:
+              `Enable the creation of index patterns which capture rollup indices,
+              which in turn enable visualizations based on rollup data.`,
```

Comment:
I think we need to keep the sentence about refreshing the page 🙂 when I disable this setting, I still get the split button option for creating a rollup index pattern until I refresh the page

Reply:
fixed